# HG changeset patch
# User Rik <rik@octave.org>
# Date 1547879447 18000
#      Sat Jan 19 01:30:47 2019 -0500
# Branch stable
# Node ID bed1d5d6dc5aaecbd719594c447244d23227b143
# Parent  49a40c5ad734d61ff6cda879a2bfa0a2e4e04760
datenum.m: Add warning about time zones and DST to documentation.

* datenum.m: Add warning about time zones and DST to documentation.

diff --git a/scripts/time/datenum.m b/scripts/time/datenum.m
--- a/scripts/time/datenum.m
+++ b/scripts/time/datenum.m
@@ -70,24 +70,32 @@
 ##
 ## @item
 ## Days before the beginning of the month go to the previous month.
 ##
 ## @item
 ## Days can be fractional.
 ## @end itemize
 ##
+## @strong{Caution:} datenums represent a specific time for the Earth as a
+## a whole.  They do not take in to account time zones (shifts in time based
+## on location), nor seasonal changes due to Daylight Savings Time (shifts in
+## time based on local regulation).  Be aware that it is possible to create
+## datenums that, when interpreted by a function which accounts for time zone
+## and DST shifts such as @code{datestr}, are nonexistent or ambiguous.
+##
 ## @strong{Caution:} this function does not attempt to handle Julian calendars
 ## so dates before October 15, 1582 are wrong by as much as eleven days.  Also,
 ## be aware that only Roman Catholic countries adopted the calendar in 1582.
 ## It took until 1924 for it to be adopted everywhere.  See the Wikipedia entry
 ## on the Gregorian calendar for more details.
 ##
 ## @strong{Warning:} leap seconds are ignored.  A table of leap seconds is
 ## available on the Wikipedia entry for leap seconds.
+##
 ## @seealso{datestr, datevec, now, clock, date}
 ## @end deftypefn
 
 ## Algorithm: Peter Baum (http://vsg.cape.com/~pbaum/date/date0.htm)
 ## Author: pkienzle <pkienzle@users.sf.net>
 
 function [days, secs] = datenum (year, month = [], day = [], hour = 0, minute = 0, second = 0)
 
